modules/report/js/commerce_pos_report.js

```diff
@@ -1,135 +0,0 @@
-(function ($) {
```

Comment:
Temporarily added alert to confirm all three tabs shows alert http://puu.sh/vYPRW/80fb03ba99.png also tried using all sort options on Journal role tab and all seems to be working nicely for me.
I actually added same css class for both the reports (Sales report and Journal report) so we can use 1 js file rather than 2 .